README.md

```diff
@@ -79,7 +79,7 @@ Please also take a deeper look into the [FAQ] - there are additional examples av
ist will be added to the authoritative policy with<br>its current value as defined in the Google Cloud Platform. Example use case: Composer 2 adds values to<br>`roles/iam.workloadIdentityUser` binding when environment is created or updated. Thus, you might want to automatically<br>import those permissions.<br><br>**`github_action_repositories` (optional):** You can list GitHub repositories (format: user/repo) here. A Workload Identity Pool and a Workload Identity Pool provider needed for Workload Identity Federation will be created automatically. Each repository given gains permissions to authenticate as this service account using Workload Identity Federation.<br>This allows any GitHub Action pipeline to use this service account without the need for service account keys.<br>An example can be found within the [FAQ].<br>For more details, see the documentation for Google's GitHub action<br>[`google-github-actions/auth`](https://github.com/google-github-actions/auth).<br><br>**Remark:** If you configure `github_action_repositories`, the module binds a member for each repository to the role<br>`roles/iam.workloadIdentityUser` inside the service account's IAM policy. This is done *regardless of whether<br>or not* you list this role in the `iam_non_authoritative_roles` key.<br><br>**Remark:** You need to grant the role `roles/iam.workloadIdentityPoolAdmin` to the principle that is<br>executing the terraform code (most likely your service account used in your pipeline) if you plan to use<br>`github_action_repositories`.<br><br>Example:<pre>service_accounts = {<br>    deployments = {<br>      display_name = "Deployments"<br>      description  = "Service Account to deploy application"<br>      description  = "<br>      iam          = {<br>        "roles/iam.serviceAccountKeyAdmin" = [<br>          "group:customer.project-role@cloudfoundation.metro.digital",<br>        ]<br>      }<br>      github_action_repositories = [<br>        "my-user-or-organisation/my-great-repo"<br>      ]<br>    }<br>    bq-reader = {<br>      display_name = "BigQuery Reader"<br>      description  = "Service Account for BigQuery Reader for App XYZ"<br>      iam          = {} # No special Service Account resource IAM permissions<br>    }<br>    composer = {<br>      display_name                = "Composer"<br>      description                 = "Service Account to run Composer 2"<br>      iam                         = {} # No special Service Account resource IAM permissions<br>      iam_non_authoritative_roles = [<br>        # maintained by Composer service - imports any existing value<br>        "roles/iam.workloadIdentityUser"<br>      ]<br>    }<br>  }<br>}</pre> | <pre>map(object({<br>    display_name                = string<br>    description                 = optional(string)<br>    iam                         = map(list(string))<br>    iam_non_authoritative_roles = optional(list(string))<br>    github_action_repositories  = optional(list(string))<br>  }))</pre> | `{}` | no |
```

Comment:
The content here is generated from the terraform code by terraform-docs (executed via pre-commit).

You need to change the description in [variables.tf](variables.tf)

Reply:
ok, that explain the poor formatting in the README. I added the changes to the variables. Still asking myself why we do it here with the hook but for the FAQ we are not doing it.